cfg/base.go

```diff
@@ -87,6 +88,16 @@ func parseOperationOptions(o []OperationOption) *OperationOptions {
 	return opts
 }
 
+func determineCandidateConfigFilename(c string) string {
+	candidateConfigFilename := c
+
+	if c != "" {
```

Comment:
Given my other comment, should it be `if c == "" {` ?

Reply:
🤣 I'm glad you're here today.... I'm falling apart :) yeah, another good call!